features/child_workflow/result/feature.java

```diff
@@ -5,52 +5,66 @@
 import io.temporal.sdkfeatures.Feature;
```

Comment:
The diff in the file is easier to understand with hide-whitespace on: spotless converting 4-space to 2-space indentation.

Reply:
Greatly appreciated! I think on the Java SDK there is an auto-formatter. I wonder if we want to bring that over here.